day01/ex02/ex02.md

```diff
@@ -0,0 +1,103 @@
+# Exercise 02 - Mutiples features and Normal Equation
```

Comment:
The mutliple features are done in the previous exo. The name is may be not appropriate anymore ?